Source/WebCore/Modules/webxr/WebXROpaqueFramebuffer.cpp

```diff
@@ -332,10 +341,8 @@ bool WebXROpaqueFramebuffer::setupFramebuffer()
                     gl.framebufferRenderbuffer(GL::FRAMEBUFFER, GL::STENCIL_ATTACHMENT, GL::RENDERBUFFER, m_depthStencilBuffer);
                 if (m_attributes.depth)
                     gl.framebufferRenderbuffer(GL::FRAMEBUFFER, GL::DEPTH_ATTACHMENT, GL::RENDERBUFFER, m_depthStencilBuffer);
-            } else {
-                if (m_attributes.stencil || m_attributes.depth)
-                    gl.framebufferRenderbuffer(GL::FRAMEBUFFER, GL::DEPTH_STENCIL_ATTACHMENT, GL::RENDERBUFFER, m_depthStencilBuffer);
-            }
+            } else
```

Comment:
We've already determined `m_attributes.stencil || m_attributes.depth` is true by now.

Reply:
Maybe assert that to catch any refractor that breaks that assumption?